ModelCatalogueGenomicsPlugin/test/integration/org/modelcatalogue/gel/GelCsvExporterSpec.groovy

```diff
@@ -1,83 +0,0 @@
-package org.modelcatalogue.gel
-
-import grails.test.spock.IntegrationSpec
-import groovy.json.JsonOutput
-import org.modelcatalogue.core.DataClass
-import org.modelcatalogue.core.DataModelService
-import org.modelcatalogue.core.ElementService
-import org.modelcatalogue.core.InitCatalogueService
-import org.modelcatalogue.core.util.builder.DefaultCatalogueBuilder
-
-/**
- * Created by dexterawoyemi on 21/03/2016.
- */
-class GelCsvExporterSpec extends IntegrationSpec {
```

Comment:
@dexterawoyemiGE is there any reason to delete the spec completely or have you just forgot to commit the renamed spec?